components/processing/form_parser/deployment/main.tf

```diff
@@ -12,6 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+locals {
+  dpu_label = {
```

Comment:
Do we want to change the variable name as well?

Reply:
It depends on level of effort. We can look at making those refactoring in v1.3 or beyond.